pkg/domain/infosync/info.go

```diff
@@ -86,6 +86,10 @@ const (
 	TopologyTiProxy = "/topology/tiproxy"
 	// infoSuffix is the suffix of TiDB/TiProxy topology info.
 	infoSuffix = "/info"
+	// TopologyTiCDC means address of TiCDC.
+	TopologyTiCDC = "/tidb/cdc"
```

Comment:
Why not `/topology/cdc` like others?

Reply:
In the implementation of tidb-dashboard, in order to avoid persistent duplicate information, we reuse the existing paths of cdc. 

But it does make more sense to use the same path convention in tidb. Thanks for your advice, and I'll fix it later.

Reply:
Do you mean TiDB-Dashboard has already used `/tidb/cdc`?
If that's true, it's up to you.